### PR TITLE
FOUC Fix

### LINKED ISSUE #76: Helmet title update causes Chrome to reload fonts when CSS file included in link attribute
My `Application` component has a Helmet reference similar to the following:

```
<Helmet
    titleTemplate='%s | My Site'
    link={[
        rel:'stylesheet', type:'text/css','href':'http://<domain>/myfont.css'}
    ]}
/>
```
and the only content within the CSS file is a few font-face declaration for the various weights of the font:
```
@font-face {
    font-family: 'MyFont';
    src: url('MyFont-Regular.eot');
    src: url('MyFont-Regular.eot?#iefix') format('embedded-opentype'),
         url('MyFont-Regular.woff') format('woff');
    font-weight: 400;
    font-style: normal;
}
@font-face {
    font-family: 'MyFont';
    src: url('MyFont-Regular-Oblique.eot');
    src: url('MyFont-Regular-Oblique.eot?#iefix') format('embedded-opentype'),
         url('MyFont-Regular-Oblique.woff') format('woff');
    font-weight: 400;
    font-style: italic;
}

@font-face {
    font-family: 'MyFont';
    src: url('MyFont-Medium.eot');
    src: url('MyFont-Medium.eot?#iefix') format('embedded-opentype'),
         url('MyFont-Medium.woff') format('woff');
    font-weight: 500;
    font-style: normal;
}
```

With this set up, any time the title updates, I see a Flash of Invisible Text (FOIT) while Chrome reloads the font. This is occurring within an infinite scrolling context where the page title updates regularly and also occurs when navigating between different routes in the app. When I hardcode the `<link>` tag with the CSS file, the FOIT doesn't happen and the type doesn't flash when the page changes. I'm only seeing this issue in Chrome; I tested in Firefox and Safari and neither showed the behavior. Is my usage incorrect or is there an underlying issue with how Chrome interacts with Helmet?

### PR DESCRIPTION
**Pull request by cwelch5**
Now will keep existing tags that are unchanged, instead of removing all and re-adding new.  Fixes #76 

Thanks @lemonCMS

### CODE REVIEW

**Comment by cwelch5** [comment]
Going to add a unit test to illustrate this bug fix.  Please standby.

File: src/test/HelmetTest.jsx [hunk 1]
```diff
@@ -1249,6 +1272,60 @@ describe("Helmet", () => {
             expect(spy.callCount).to.equal(1);
         });
 
+        it("will only add new tags and will perserve tags when rendering additional Helmet instances", () => {
```
Comment (potench) [inline src/test/HelmetTest.jsx]: +1

File: src/test/HelmetTest.jsx [hunk 2]
```diff
@@ -157,12 +157,34 @@ describe("Helmet", () => {
...
-                const args = spy.getCall(0).args[0];
-                expect(args).to.contain({title: "Main Title"});
-                expect(args.baseTag).to.contain({href: "http://mysite.com/"});
-                expect(args.metaTags).to.contain({"charset": "utf-8"});
-                expect(args.linkTags).to.contain({"href": "http://localhost/helmet", "rel": "canonical"});
-                expect(args.scriptTags).to.contain({"src": "http://localhost/test.js", "type": "text/javascript"});
+                const newState = spy.getCall(0).args[0];
+                const addedTags = spy.getCall(0).args[1];
+                const removedTags = spy.getCall(0).args[2];
+                debugger;
```
Comment (potench) [inline src/test/HelmetTest.jsx]: remove `debugger`

Comment (cwelch5) [inline src/test/HelmetTest.jsx]: dang, nice catch..phew

**Pull request merged**